dashboard/app/controllers/application_controller.rb

```diff
@@ -129,6 +129,10 @@ def milestone_response(options)
         script_level_solved_response(response, script_level)
       else # not solved
         response[:message] = 'try again'
+        response[:hint_view_requests] = HintViewRequest.milestone_response(script_level.script, script_level.level, current_user)
```

Comment:
what does this do if there is no logged in user? should this whole thing even be called?